Cargo.lock

```diff
@@ -2048,9 +2048,9 @@ dependencies = [
 
 [[package]]
 name = "rustls"
-version = "0.23.12"
+version = "0.23.13"
```

Comment:
This is actually pretty nice because aws-lc-rs now comes with optional prebuilt ASM, which rustls enables by default.